crates/torii/grpc/src/server/mod.rs

```diff
@@ -132,7 +132,7 @@ impl DojoWorld {
 impl DojoWorld {
     pub async fn metadata(&self) -> Result<proto::types::WorldMetadata, Error> {
         let world_address = sqlx::query_scalar(&format!(
-            "SELECT world_address FROM worlds WHERE id = '{:#x}'",
+            "SELECT contract_address FROM contracts WHERE id = '{:#x}'",
```

Comment:
Why should we fetch if the world address is already inside `self`?

Reply:
honestly not sure, but i assumed it would be required incase `id` is not world address so kept it

Reply:
Let's keep it like that for now then, thanks for the answer.